hs_core/templates/search/search.html

```diff
@@ -631,6 +606,16 @@ <h4 class="panel-title">
                 }
             });
 
+            for (var  i = 0; i < json_results.length; i++ ) {
+                if (json_results[i].coverage_type == 'point') {
+                    checkDuplicatePointResults(modified_points_data, json_results[i]);
+
+                }
+                else if(json_results[i].coverage_type == 'box') {
+                    checkDuplicateBoxResults(modified_boxes_data, json_results[i]);
+
+                }
+            }
```

Comment:
@mauriel133 We have the same forloop to add resources to markers twice in this function. It would cause each marker has duplicated resources.
![image](https://cloud.githubusercontent.com/assets/6678333/16467809/04041ce4-3e17-11e6-8161-e4aa3265d2f9.png)

We can remove this one, and keep the one above.